atlp-rwanda/orcas-phantom-frontend
# ft: improving the landing page

### What this PR does

- Improving the UI of the landing page for both mobile and web view

### Task to be completed

- Add two separate logos for both mobile and web view

- Add hover effects to nav links

- improve the UI of the Sidemenu of mobile view by removing the top white area

### Manual testing

- Clone the github repository


- run npm install


- run npm run dev to start the dev server


- check mobile and web view of `http://localhost:8080/home`


### Background context
N/A

### Relevant pivotal tracker stories
N/A

### Screenshots
![image](https://user-images.githubusercontent.com/54126307/98921976-9408b080-24da-11eb-836b-04f21744232c.png)


![image](https://user-images.githubusercontent.com/54126307/98922052-b00c5200-24da-11eb-81dd-fd14a7ce2648.png)


![image](https://user-images.githubusercontent.com/54126307/98922097-bd294100-24da-11eb-8629-b42899f54cd0.png)
